src/resources/app-en.json

```diff
@@ -1100,7 +1100,9 @@
     "respondents": "Respondents",
     "subjects": "Subjects",
     "submissions": "Submissions",
-    "currentlyAssigned": "Currently Assigned"
+    "currentlyAssigned": "Currently Assigned",
+    "lastSubmission": "Last activity:",
+    "noDataYet": "No data yet"
```

Comment:
What about Greek language translations?

Reply:
Ah good call. Need to make this a new habit!

Reply:
Oh. Actually this PR is for the admin app, which doesn't have support for Greek yet. Not sure if/when it's planned for addition.

Reply:
Ah right, no Greek in admin 🤦